financial-connections/src/main/java/com/stripe/android/financialconnections/ui/FinancialConnectionsSheetNativeActivity.kt

```diff
@@ -162,4 +181,13 @@ internal class FinancialConnectionsSheetNativeActivity : AppCompatActivity(), Ma
             }
         }
     }
+
+    override fun onDestroy() {
+        super.onDestroy()
+        imageLoader.cancel()
```

Comment:
Can we avoid the need to call `cancel()` manually? Maybe with a lifecycle observer that automatically cancels the loader?

Reply:
Added `lifecycle` parameter, and triggered cancel on destroy - https://github.com/stripe/stripe-android/pull/5307/commits/655a2b6f5f53a8902e055d7d225c8e3a633c640d

StripeImage would still call `cancel` when disposed, that will cancel the ongoing image fetch, if any.